pydantic/typing.py

```diff
@@ -399,7 +399,10 @@ def resolve_annotations(raw_annotations: Dict[str, Type[Any]], module_name: Opti
             else:
                 value = ForwardRef(value, is_argument=False)
         try:
-            value = _eval_type(value, base_globals, None)
+            if sys.version_info > (3, 13):
```

Comment:
Shouldn't it be `>= 3.13`?
```suggestion
            if sys.version_info >= (3, 13):
```